### PR TITLE
New package: vmware-horizon-client-2303

### PR DESCRIPTION
**Pull request by abenson**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

**Comment by abenson** [comment]
I'm pretty happy with the state of `vmware-horizon-client`.  It and the `-smartcard` sub-package works well with my company's setup, and I care less about it being pretty since it's a restricted package anyway.  I haven't tested the USB redirection since we have that disabled by policy.

I would like the `libudev0-shim` to be more better before its merged, but I'm not sure how best to fix it.  In `do_build`, I had to unset `LDFLAGS` to get rid of the `-Wl,--as-needed` otherwise it won't pull in `libudev.so.1` and it'll fail on load.

**Comment by abenson** [comment]
I guess I could try getting the arm64 version to work, too.

File: srcpkgs/vmware-horizon-client/files/vmware-USBArbitrator/run [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+exec 2>&1
+
+exec chpst -P /usr/bin/vmware-usbarbitrator
```
Comment (Johnnynator) [inline srcpkgs/vmware-horizon-client/files/vmware-USBArbitrator/run:5]: You want to add the `-f` flag at the end to run it in the foreground.
After that the service runs fine, but the usb support in vmware horizon still seems to be broken.

Comment (abenson) [reply srcpkgs/vmware-horizon-client/files/vmware-USBArbitrator/run]: Fixed, and USB seems to be working for me!

**Pull request merged**